lib/async_hooks.js

```diff
@@ -49,6 +49,9 @@ const init_symbol = Symbol('init');
 const before_symbol = Symbol('before');
 const after_symbol = Symbol('after');
 const destroy_symbol = Symbol('destroy');
+const emitBeforeN = emitHookFactory(before_symbol, 'emitBeforeN');
```

Comment:
Could you change the `N` to `Native` while you are at it? That short naming convention really annoys me.